apps/web/src/app/_components/footer.tsx

```diff
@@ -1,9 +1,10 @@
 import Image from "next/image";
 import Link from "next/link";
+import { MessageSquare } from "lucide-react";
 
 export default function Footer() {
   return (
-    <footer className="flex h-20 w-full grid-cols-2 items-center justify-between border-t border-t-[#919191] bg-white px-4">
+    <footer className="flex h-20 w-full grid-cols-2 flex-col items-center justify-between border-t border-t-[#919191] bg-white px-4 md:flex-row">
```

Comment:
Ok so I played around with spacing a bit and I think we should get rid of the `h-20` and just replace the `px-4` with `p-4`. This way everything is like evenly spaced and what not.

Also like we did below let's also add `gap-2` here to keep things spaced out